src/plugins/intel_gpu/src/graph/softmax.cpp

```diff
@@ -31,6 +31,11 @@ std::string softmax_inst::to_string(softmax_node const& node) {
 
     std::stringstream primitive_description;
 
+    json_composite softmax_info;
+    softmax_info.add("input id", node.input().id());
```

Comment:
I think it's not needed here. We should either use "dependencies" field to track inputs in the dump or add dependencies ids dumping into the common code for all primitives.